recipe/meta.yaml

```diff
@@ -33,13 +33,10 @@ requirements:
     - pyyaml
   run_constrained:
     # modin
-    - pandas >=1.0.0,<3.0.0
     - modin ==0.28.1
     # opentelemetry
     - opentelemetry-api >=1.0.0,<2.0.0
     - opentelemetry-sdk >=1.0.0,<2.0.0
-    # secure-local-storage
-    - keyring >=23.1.0,<26.0.0
```

Comment:
you can leave keyring too because of https://github.com/snowflakedb/snowpark-python/blob/bb1ed3d5e618f3aa5744c7c5ca88b6b14bb4d5b2/setup.py#L114-L116

for `snowflake-connector-python[secure-local-storage]` as in https://inspector.pypi.io/project/snowflake-connector-python/3.12.2/packages/5e/78/7e3ce98acef501ab3509958ed8f95e31720e225b7f478b7d7e70ff731fc9/snowflake_connector_python-3.12.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl/snowflake_connector_python-3.12.2.dist-info/METADATA#line.76